cmd/svcat/instance/deprovision_cmd.go

```diff
@@ -68,7 +88,30 @@ func (c *deprovisonCmd) Run() error {
 }
 
 func (c *deprovisonCmd) deprovision() error {
-	err := c.App.Deprovision(c.Namespace, c.instanceName)
+	var err error
+	if c.abandon {
+		fmt.Fprintln(c.Output, "This action is not reversible and may cause you to be charged for the broker resources that are abandoned.")
+		if !c.skipPrompt {
+			fmt.Fprintln(c.Output, "Are you sure? [y|n]: ")
+			s := bufio.NewScanner(os.Stdin)
+			s.Scan()
+
+			err = s.Err()
+			fmt.Fprintln(c.Output, err)
+
+			if strings.ToLower(s.Text()) != "y" {
+				err = fmt.Errorf("aborted abandon operation")
+				return err
+			}
+		}
+
+		si := &v1beta1.ServiceInstance{ObjectMeta: metav1.ObjectMeta{Name: c.instanceName, Namespace: c.Namespace}}
+		if _, err = c.App.RemoveBindingFinalizerByInstance(si); err != nil {
```

Comment:
As we discussed in SIG meeting on Monday, this is the problem.  We need to remove the finalizers from this **instance**.  This call to RemoveBindingFinalizerByInstance is removing the finalizer from any **bindings** related to the instance.